lib/services/ios-project-service.ts

```diff
@@ -655,12 +711,16 @@ We will now place an empty obsolete compatability white screen LauncScreen.xib f
 	}
 
 	public async prepareProject(projectData: IProjectData, platformSpecificData: IPlatformSpecificData): Promise<void> {
-		let provision = platformSpecificData && platformSpecificData.provision;
+		const projectRoot = path.join(projectData.platformsDir, "ios");
```

Comment:
The "ios" here is the name of the directory in "platforms" where the native project is created. Do we have a constant in the project service or something that is not somewhat related with devices? As I look into the IDevicePlatformsConstants I wonder why there is a "Device" in the name?